backup-restore/hotfixes/2.8.0/applypatch-280.sh

```diff
@@ -59,10 +59,11 @@ oc set data -n $BR_NS cm/guardian-configmap datamoverJobpodEphemeralStorageReque
 oc set data -n $BR_NS cm/guardian-configmap datamoverJobpodMemoryRequest='4000Mi'
 oc set data -n $BR_NS cm/guardian-configmap datamoverJobpodMemoryRequestRes='4000Mi'
 
+SKIP_MINIO="true"
```

Comment:
Any good reason why we want to keep it anyway?
I would prefer to just remove MINIO related code.

Reply:
The latest minio image has the fix for this issue: Deleted minio objects are not deleted from minio PVC. 
Users that run into this bug might prefer to upgrade even if it will cause problems with the next upgrade of Backup and Restore service.

Reply:
In that case our first preference should be  convince customer to upgrade to 2.8.2 and then apply 2.8.2 hotfix. 
If for some strong reason customer can not upgrade, we should handle that as one off, rather than giving hint to all customers.